velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -749,6 +790,51 @@ std::string PrestoQueryRunner::createTable(
   return tableDirectoryPath;
 }
 
+std::unordered_map<std::string, std::vector<velox::RowVectorPtr>>
+PrestoQueryRunner::getAllTablesAndNames(const core::PlanNodePtr& plan) {
```

Comment:
same here. Should this be part of the base class?

Reply:
Done.